### ISSUE TITLE
Layout/HashAlignment for method last argument, align as other method arguments

### ISSUE THREAD

**Issue by vfonic** [rubocop/rubocop#13736]
## Is your feature request related to a problem? Please describe.

I want to treat all method arguments the same, whether they are "last argument"/options/kwargs/... or not.

Currently, Layout/HashAlignment treats all of the remaining last/hash arguments separately from the other method arguments.

For example:

```ruby
# bad 
f.input :records_type, as: :select, input_html: { disabled: !f.object.new_record? },
                       collection: [RecordsType::EVENTS, RecordsType::PLACES, RecordsType::STORIES].map { [_1.humanize, _1] }

# good
f.input :records_type, as: :select, input_html: { disabled: !f.object.new_record? },
        collection: [RecordsType::EVENTS, RecordsType::PLACES, RecordsType::STORIES].map { [_1.humanize, _1] }
```

## Describe the solution you'd like

I'd prefer to be able to set the second ("good") arguments alignment.

## Describe alternatives you've considered

I've looked into available options for this cop:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/HashAlignment
`EnforcedHashRocketStyle `, `EnforcedColonStyle`, `EnforcedLastArgumentHashStyle`

I've also looked into how to achieve this by using: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/ArgumentAlignment

Seems like if I disable `Layout/HashAlignment` cop, then last hash argument is not aligned/checked at all. `Layout/ArgumentAlignment` still works for all the other non-hash arguments. This seems odd. Would it make sense to move the logic for "last hash argument" from `Layout/HashAlignment` to `Layout/ArgumentAlignment`?

**Comment by dvandersluis** [comment]
Using this configuration gives me the indentation you're looking for:

```yaml
Layout/HashAlignment:
  Enabled: true
  EnforcedHashRocketStyle: key
  EnforcedColonStyle: key
  EnforcedLastArgumentHashStyle: always_inspect

Layout/ArgumentAlignment:
  Enabled: true
  EnforcedStyle: with_first_argument
```

That converts

```ruby
f.input :records_type, as: :select, input_html: { disabled: !f.object.new_record? },
  collection: [RecordsType::EVENTS, RecordsType::PLACES, RecordsType::STORIES].map { [_1.humanize, _1] }
```

to 

```ruby
f.input :records_type, as: :select, input_html: { disabled: !f.object.new_record? },
                       collection: [RecordsType::EVENTS, RecordsType::PLACES, RecordsType::STORIES].map { [_1.humanize, _1] }
```

**Comment by vfonic** [comment]
That's the opposite (and default configuration) of what I want:

> I'd prefer to be able to set the first ("bad") arguments alignment.
